content/en/docs/guides/custom-backend-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Custom Backend Metrics
+description : >-
+  A mechanism in gRPC library that allows users to inject custom metrics at the
+  gRPC server side (backends). These metrics are then reported to the gRPC
+  clients where they can be consumed by the custom load balancing algorithms to
+  make traffic distribution decisions.
+---
+
+
+### Overview
+Simple load balancing decisions can be made by taking into account local or
+global knowledge of a backend’s load, for example CPU. More sophisticated
+loadbalancing decisions are possible with application specific knowledge, e.g.
+queue depth, or by combining multiple metrics.
+
+The custom backend metrics feature exposes APIs to allow users to implement
+the metrics feedback in their LB policies.
+
+### User Cases
+The feature is mainly for advanced uses cases where a custom LB policy is used
```

Comment:
I don't know how I feel about calling something an "advanced use case." This is really a value judgment that we as the library developers/maintainers might not be qualified to make. It often has the effect of discouraging people from using it.